gnosis/gp-ui
# Order not found UI

# Summary
Closes #641 
Create a Order not found page result.
Design:
https://www.figma.com/file/m7obOydxAhGNmlFYvxTk3Z/GP-Explorer?node-id=1144%3A1416

# To Test
1. Search for example: "0xb6BAd41ae76A11D10f7b0E664C5007b908bC77C9" in the [order box search](https://pr658--gpui.review.gnosisdev.com/) (home).
2. It should display the ["Order not found" page](https://pr658--gpui.review.gnosisdev.com/orders/0xb6BAd41ae76A11D10f7b0E664C5007b908bC77C9). 

![image](https://user-images.githubusercontent.com/622217/134690973-3c038fd9-034d-4223-8196-3868e5685e95.png)

![image](https://user-images.githubusercontent.com/622217/134691021-dc97f2a6-4031-4e2a-8c17-0c0310895f9d.png)
